### ISSUE TITLE
Post pictures?

### ISSUE THREAD

**Issue by SpeedGP** [Bumblefuck/25mm-olfa-blade-holder-knife#1]
Can we have a place to post pictures of our deranged creations and battle scars?

**Comment by Bumblefuck** [comment]
No. It would be cool, but then it needs to be tended to and I got a full plate.

**Comment by SpeedGP** [comment]
![foto_no_exif (1)](https://github.com/user-attachments/assets/8a83296a-a2c8-41e8-9282-f477764f264a)

**Comment by Bumblefuck** [comment]
Oh snap! Glow in the dark. Brilliant! (hurr-hurr)

**Comment by mxlje** [comment]
I finally managed to pick up some blades but I’m wondering if

a) my blades are longer than yours or
b) it needs one more notch in the back for a proper "closed" position?

It works great but the closed position is a bit weird with the spring being compressed so much and it also makes it somewhat easy to pull it all the way out when closing it quickly.

![closed](https://github.com/user-attachments/assets/cd190193-96de-4958-b8ef-7d98af2d3586)
_fully closed but blade sticking out_

![closed](https://github.com/user-attachments/assets/27113a29-bfa5-420c-913f-1def97445b3f)
_closed one step more_

![end](https://github.com/user-attachments/assets/bd004527-1a38-4028-8c65-27f3b992a622)
_spring fit is off_

Edit: I know this will become a non-issue once I break off the first part of the blade but that’s not really the point.


@Bumblefuck any thoughts?

**Comment by SpeedGP** [comment]
That print is CLEAN! I was thinking the same thing about the back not being closed..... I'm totally going to cut myself at some point.

The blade looks to be the same length as mine. I get what you mean, but I think if the spring ever wears out, just print a new one.

I ended up using a little bit of SuperLube to help lessen friction and wear it in a bit
